script/update-archive-version.ts

```diff
@@ -35,22 +59,26 @@ async function updateArchiveVersion() {
   );
 
   const newBookLoader = makeUnifiedBookLoader(
-    createArchiveLoader(args.newArchive, {
+    createArchiveLoader(`${REACT_APP_ARCHIVE_URL_BASE}${args.pipelineVersion}`, {
```

Comment:
since the other script is no longer callable, shouldn't this pipeline version default to the current version to handle the situation where you just want to update content with no pipeline change?